qa/os/src/test/java/org/elasticsearch/packaging/test/DebPreservationTests.java

```diff
@@ -63,10 +63,6 @@ public void test20Remove() throws Exception {
             installation.config("users_roles")
         );
 
-        // keystore was removed
-
-        assertPathsDoNotExist(installation.config("elasticsearch.keystore"), installation.config(".elasticsearch.keystore.initial_md5sum"));
```

Comment:
Should we update the statement above to check that they keystore still exists?

Reply:
Yes, makes sense - I was too quick to remove coverage :/